src/views/projectExplorer/index.ts

```diff
@@ -1066,9 +1066,10 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
       }),
       commands.registerCommand(`vscode-ibmi-projectexplorer.runAction`, async (element: Project | ObjectFile | MemberFile) => {
         if (element) {
+          let workspaceFolder;
           let chosenUri = element.resourceUri;
 
-          // if project, get the uri of the active editor file if appropriate
+          // If project, get the uri of the active editor file if appropriate
```

Comment:
I don't see the workspaceFolder being set if the condiction at 1073 is satisfied.  Is it not necessary in this case?

Reply:
Yes, it is not needed in this case because when it hits line 1073, it means that this was invoked by the `Run Action` button on the project tree item. In this scenario, `chosenUri` will either be the `uri` of the active editor or the workspace folder if no active editor. Since in both these cases, we are dealing with a local path, Code4i's run action logic will know to use local `LIBL` and `CURLIB`.

This PR fixes when `Run Action` is used on objects and members since these are not local paths and so Code4i will not know what project it is associated with.